src/libs/ajax/Metrics.ts

```diff
@@ -31,14 +31,17 @@ export const Metrics = signal => ({
     }
 
     return fetchBard('api/event', _.mergeAll([isRegistered ? authOpts() : undefined, jsonBody(body), { signal, method: 'POST' }]))
-  }),
+  }
+  return ({
+    captureEvent: (withErrorIgnoring(captureEventFn) as unknown as typeof captureEventFn),
```

Comment:
Why do we need to do the unknown cast along the way? I've mostly seen that coercing between different generated objects of the same type (AoU Angular->React conversion)

Reply:
this is always a last resort in my book... the hoF withErrorIgnoring had a bit too much lodash-fp type wiggliness to be cooperative without expanding the scope of the code changes into converting the related bits into TS as well.